sound/soc/sof/intel/hda.c

```diff
@@ -71,6 +71,11 @@ static u32 hda_get_interface_mask(struct snd_sof_dev *sdev)
 				    BIT(SOF_DAI_INTEL_HDA) | BIT(SOF_DAI_INTEL_ALH);
 		interface_mask[1] = BIT(SOF_DAI_INTEL_HDA);
 		break;
+	case SOF_INTEL_ACE_2_0:
+		interface_mask[0] = BIT(SOF_DAI_INTEL_SSP) | BIT(SOF_DAI_INTEL_DMIC) |
+				    BIT(SOF_DAI_INTEL_HDA) | BIT(SOF_DAI_INTEL_ALH);
+		interface_mask[1] = interface_mask[0]; /* all interfaces accessible without DSP */
```

Comment:
Well, it depends on what the definition of 'interface_mask' is... If this describes the hardware capabilities the code is correct. If this describes whether this interface will work out of the box, then it's not.

But in the latter case, we have a similar problem with HDaudio anyways. You need a custom topology without mixers, otherwise things can break. 

I don't really care about this code, but we need to agree on the semantics on the interface_mask @ujfalusi @ranj063 @kv2019i